v0/destinations/hs/config.js

```diff
@@ -1,18 +1,60 @@
 const { getMappingConfig } = require("../../util");
 
+const BASE_ENDPOINT = "https://api.hubapi.com";
+
+const CONTACT_PROPERTIES = `${BASE_ENDPOINT}/properties/v1/contacts/properties`;
```

Comment:
`FETCH_CONTACT_PROPERTIES_ENDPOINT` is more intuitive

Reply:
This endpoint can be used for POST, GET, etc. Prefixing it with FETCH will be specific to GET

Reply:
Then atleast `CONTACT_PROPERTY_MAP_ENDPOINT`